src/python/grapl-common/grapl_common/debugger/vsc_debugger.py

```diff
@@ -0,0 +1,90 @@
+"""
+I arbitrarily chose `debugpy`, the VS Code debugger (formerly known as ptsvd).
+It'd be pretty easy to add the PyCharm/IntelliJ debugger too (which uses pydevd)
+"""
+import subprocess
+import sys
+import logging
+import os
+
+
+def _install_from_pip(package: str) -> None:
+    # Gross, but the suggested way to install from pip mid-python-program!
+    # Doing it this way so that <every executable that depends on grapl_common> doesn't inherently
+    # need to import debugpy
+    subprocess.check_call(
+        [sys.executable, "-m", "pip", "install", "--upgrade", package]
+    )
```

Comment:
this here is, by far, the most controversial thing in this diff. basically declaring a _runtime dependency_

there's nothing inherently bad about each executable including `debugpy` - there's just no reason that a release version of any of these services, on prod, would ever need that library

Reply:
👍